microsoft-edge/webview/PermissionRequest.md

```diff
@@ -1,30 +1,140 @@
----
-description: Provides event information about the current permission request
-title: PermissionRequest object
-author: libbymc
-ms.author: libbymc
-ms.date: 02/12/2018
-ms.topic: reference
-ms.prod: microsoft-edge
-keywords: webview, windows 10 apps, uwp, edge
----
-
-# PermissionRequest object
-
-Provides event information about the current permission request.
-
-## Properties
-
-### permissionRequest
-
-Returns a **PermissionRequest** object that represents the end-user permission request made by content of the [webview](../webview.md).
-
-This property is read-only.
-
-```js
-var webView = document.getElementById("myWebView");
-webView.addEventListener("MSWebViewPermissionRequested", function(e) {
-    var permissionRequest = e.permissionRequest;
-    // Examine and/or defer permission request
-}, false);
-```
+---
+description: Provides information about a permission request
+title: PermissionRequest object
+author: libbymc
+ms.author: libbymc
+ms.date: 02/12/2018
+ms.topic: reference
+ms.prod: microsoft-edge
+keywords: webview, windows 10 apps, uwp, edge
+---
+
+# PermissionRequest object
+
+Provides information about a permission request. This object is available from the permissionRequest property of the event args from the [MSWebViewPermissioniRequested](../webview.md#MSWebViewPermissionRequested) webview event.

```

Comment:
This is the illegal link the PR tooling mentioned. Need to figure out what the correct way to do this is.

Reply:
Just make the link lower case (../webview.md##mswebviewpermissionrequested) and that should work!